src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1333,36 +1336,38 @@ internal static bool IsCustomTaskType(this NamedTypeSymbol type, out NamedTypeSy
         /// </summary>
         internal static TypeSymbol NormalizeTaskTypes(this TypeSymbol type, CSharpCompilation compilation)
         {
-            NormalizeTaskTypesCore(compilation, ref type);
+            NormalizeTaskTypesInType(compilation, ref type);
             return type;
         }
 
         /// <summary>
         /// Replace Task-like types with Task types. Returns true if there were changes.
         /// </summary>
-        private static bool NormalizeTaskTypesCore(CSharpCompilation compilation, ref TypeSymbol type)
+        private static bool NormalizeTaskTypesInType(CSharpCompilation compilation, ref TypeSymbol type)
         {
             switch (type.Kind)
```

Comment:
I am not sure I am following. Can array type be task-like?

---
In reply to: [74288645](https://github.com/dotnet/roslyn/pull/12671#discussion_r74288645) [](ancestors = 74288645,74285338)